code/modules/surgery/healing.dm

```diff
@@ -113,6 +118,33 @@
 	desc = "A surgical procedure that provides experimental treatment for a patient's brute traumas. Heals considerably more when the patient is severely injured."
 
 /********************BRUTE STEPS********************/
+/datum/surgery_step/heal/brute/get_progress(mob/user, mob/living/carbon/target, brute_healed, burn_healed)
+	if(!brute_healed)
+		return
+
+	var/estimated_remaining_steps = target.getBruteLoss() / brute_healed
+	var/progress_text
+	if(locate(/obj/item/healthanalyzer) in user.held_items)
+		progress_text = ". Remaining brute: <font color='#ff3333'>[target.getBruteLoss()]</font>"
+	else
+		switch(estimated_remaining_steps)
```

Comment:
Drop this into its own proc, so if it needs updated in the future, it won't need updated three times

Reply:
i'll figure out how to do that eventually

Reply:
Make a proc, call it something like `get_steps_message` that takes a number, and returns the string associated with that many steps remaining

Reply:
wouldn't it be three different procs then? each surgery has different strings depending on if its brute, burn, or combo

Reply:
That is what I get for not reading properly, I thought it was the same messages